files/build_templates/sonic_debian_extension.j2

```diff
@@ -723,6 +725,9 @@ sudo cp $IMAGE_CONFIGS/backend_acl/backend-acl.service $FILESYSTEM_ROOT_USR_LIB_
 sudo cp $IMAGE_CONFIGS/backend_acl/backend_acl.py $FILESYSTEM_ROOT/usr/bin/backend_acl.py
 echo "backend-acl.service" | sudo tee -a $GENERATED_SERVICE_FILE
 
+# Copy RPS script file
+sudo cp $IMAGE_CONFIGS/rps/rps.py $FILESYSTEM_ROOT/usr/bin/rps.py
```

Comment:
Any chmod changes to the file with execute permission?

Reply:
Oddly enough, it worked just fine without an explicit chmod. I'll add it anyways just to be sure.